Firestore/Source/Public/FirebaseFirestore/FIRVectorValue.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(VectorValue)
+@interface FIRVectorValue : NSObject
+
```

Comment:
```suggestion
@property(readonly) NSArray<NSNumber *> *array NS_REFINED_FOR_SWIFT;

```
```

Reply:
I think `array` is a concise and accurate name, but I'm not sure if there are next steps to expose the vector as an array of other types (floats, int, etc).

Reply:
Implemented. Using name `array` across the objective-c and Swift implementation.